example/java/io/orkes/conductor/sdk/examples/TaskRunner.java

```diff
@@ -55,7 +55,8 @@ public static void main(String[] args) {
                         .withSleepWhenRetry(
                                 500) // Time in millis to sleep when retrying for a task update.
                         // Default is 500ms
-                        .withTaskPollTimeout(100) // Poll timeout for long-poll.  Default is 100ms
+                        // .withTaskPollTimeout(100) // Poll timeout for long-poll.  Default is
```

Comment:
Question: - We don't need poll here?

Reply:
I think it's just removing a property set, due to the property being removed internally

Reply:
yes, @gardusig is correct here.